### PR TITLE
docs: Show the code of adding devtools extension

### PR DESCRIPTION
**Pull request by moshfeu**
#### Description of Change
Just to show the path is not enough because it's not working - https://github.com/nodejs/node/issues/684#issuecomment-240849243.
Replaced with a working code.

#### Checklist

- [x] PR description included (<s>and stakeholders cc'd</s> I don't know who they are)
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] PR release notes describe the change in a way relevant to app-developers

#### Release Notes

Notes: adding devtools extension docs improvement

### CODE REVIEW

File: docs/tutorial/devtools-extension.md [hunk 1]
```diff
@@ -29,7 +29,14 @@ Using the [React Developer Tools][react-devtools] as example:
    * on macOS it is `~/Library/Application Support/Google/Chrome/Default/Extensions`.
 1. Pass the location of the extension to `BrowserWindow.addDevToolsExtension`
    API, for the React Developer Tools, it is something like:
-   `~/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/0.15.0_0`
+   ```javascript
+   const { join } = require('path');
+   const { homedir } = require('os');
+
+   BrowserWindow.addDevToolsExtension(
+      join(homedir(),'/Library/Application\Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/0.15.0_0')
```
Comment (zcbenz) [inline docs/tutorial/devtools-extension.md]: The `Application\Support` should be `Application Support`.

**Comment by zcbenz** [comment]
The failing tests are not related to this PR, they can be ignored.

**Comment by moshfeu** [comment]
Thanks @zcbenz !

File: docs/tutorial/devtools-extension.md [hunk 2]
```diff
@@ -29,7 +29,14 @@ Using the [React Developer Tools][react-devtools] as example:
    * on macOS it is `~/Library/Application Support/Google/Chrome/Default/Extensions`.
 1. Pass the location of the extension to `BrowserWindow.addDevToolsExtension`
    API, for the React Developer Tools, it is something like:
-   `~/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/0.15.0_0`
+   ```javascript
+   const { join } = require('path');
+   const { homedir } = require('os');
+
+   BrowserWindow.addDevToolsExtension(
+      join(homedir(),'/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/0.15.0_0')
```
Comment (miniak) [inline docs/tutorial/devtools-extension.md]: Explicit `path.join` and `os.homedir` is more readable

**Comment by moshfeu** [comment]
@miniak Done. I think that the fail tests is not related to my change.

**Comment by moshfeu** [comment]
@zcbenz can you take a look please?

**Pull request merged**